### PR TITLE
Leaderboard: Using the leaderboard API

### PR DESCRIPTION
**Pull request by ha-manel**
## In this branch I: 
- Created a game using [the leaderboard API](https://www.notion.so/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3).
- Used the game ID generated by the API to add new scores.
- Used the game ID generated by the API to display the leaderboard.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,65 @@
...
+// get list of scores
+const scoresList = document.querySelector('#leaderboard-list ul');
+
+async function getScoresList() {
+  await fetch(requestURL)
+    .then((response) => response.json())
+    .then((json) => {
+      scoresList.innerHTML = `${json.result.sort((a, b) => b.score - a.score).map((score) => `<li class="score">${score.user}: ${score.score}</li>`).join('')}`;
+    });
+}
```
Comment (TedLivist) [inline src/index.js:57]: - [ ] In keeping with ES6 syntax requirement, kindly refactor this function into an arrow function.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,65 @@
...
+const name = document.querySelector('#name-input');
+const score = document.querySelector('#score-input');
+const form = document.querySelector('form');
+const inputs = document.querySelectorAll('input');
+const confirmationMsg = document.querySelector('#confirmation-msg');
+
+async function addNewScore() {
+  await fetch(requestURL, {
+    method: 'POST',
+    body: JSON.stringify({
```
Comment (TedLivist) [inline src/index.js:17]: - [ ] Kindly refactor to an arrow function.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @ha-manel,

Good job so far!
✔️ Good documentation
✔️ POST and GET API implementation
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @ha-manel,

You've done well working on the concerns from the previous review 👏🏼.
Tasks for this project are now successfully implemented 💯.

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**